### PR TITLE
Creating the NewHelp site with a default Jekyll installation

### PR DESCRIPTION
**Pull request by quinthar**
### Details
Implements step 2.1 of [New.Help.Expensify.com](https://docs.google.com/document/d/1OlftnRPEKHynzbOvKmkWHI-MSlPvF_a-TXSVyQCVs_w/edit#heading=h.jmfvcfvil8pc)

### Fixed Issues
$https://github.com/Expensify/Expensify/issues/414789

### Tests
1. Verify that Jekyll runs and creates the site
2. Verify that the site loads on localhost
3. Verify newhelp.expensify.com load a default Jekyll site
4. Verify help.expensify.com loads and is unaffected

### Offline tests
n/a

### QA Steps
n/a

### PR Author Checklist
- [X] I linked the correct issue in the `### Fixed Issues` section above
- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [X] I verified that all the inputs inside a form are aligned with each other.
    - [X] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.

### Screenshots/Videos
image

### CODE REVIEW

**Comment by quinthar** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by quinthar** [comment]
recheck

**Comment by quinthar** [comment]
Hey @DylanDylann I'm reassigning this to @coleaeason as he's the one who has written up the instructions I'm following now.

**Review by coleaeason — DISMISSED** [review]
looks good! nab b/c we're going to delete the default pages, but could change `.markdown` to `.md` to match our normal conventions on file naming.

**Comment by coleaeason** [comment]
## Reviewer Checklist

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is either coming verbatim from figma or has been approved by marketing (in order to get marketing approval, ask the Bug Zero team member to add the Waiting for copy label to the issue)
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/utils/index.ts) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(theme.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If the PR modifies the UI (e.g. new buttons, new UI components, changing the padding/spacing/sizing, moving components, etc) or modifies the form input styles:
    - [ ] I verified that all the inputs inside a form are aligned with each other.
    - [ ] I added `Design` label and/or tagged `@Expensify/design` so the design team can review the changes.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>



</details>

<details>
<summary>Android: mWeb Chrome</summary>



</details>

<details>
<summary>iOS: Native</summary>



</details>

<details>
<summary>iOS: mWeb Safari</summary>



</details>

<details>
<summary>MacOS: Chrome / Safari</summary>



</details>

<details>
<summary>MacOS: Desktop</summary>



</details>

File: help/Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+      google-protobuf (~> 4.27)
+    sass-embedded (1.79.3-x86_64-linux-musl)
+      google-protobuf (~> 4.27)
+    terminal-table (3.0.2)
+      unicode-display_width (>= 1.1.1, < 3)
+    unicode-display_width (2.6.0)
+    webrick (1.8.2)
+
+PLATFORMS
+  aarch64-linux
```
Comment (roryabraham) [inline help/Gemfile.lock:161]: I'm not sure why so many platforms here ... none of our other `Gemfile.locks` have so many platforms. Not sure if it matters though 🤷🏼

File: .github/workflows/deployNewHelp.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'help/**'
+  pull_request:
```
[the remark this answers is not in the document]
Comment (quinthar) [reply .github/workflows/deployNewHelp.yml:7]: My understanding is Cloudflare detects the push to main and mirrors over the content automatically -- so we don't need to explicitly push to Cloudflare.

**Comment by quinthar** [comment]
Nice! I got it working!!!  Thanks for your help @coleaeason and @roryabraham , taking off HOLD.  Can you please approve again?

**Comment by quinthar** [comment]
Hm, but it's not passing this validate test:

```
Linting workflows...
workflows/deployNewHelp.yml:71:9: shellcheck reported issue in this script: SC2086:info:1:1: Double quote to prevent globbing and word splitting [shellcheck]
   |
71 |         run: "$HOME/.local/bin/cli4 --verbose --delete 'hosts=[\"newhelp.expensify.com\"]' /zones/:9ee042e6cfc7fd45e74aa7d2f78d617b/purge_cache"
   |         ^~~~
Error: Process completed with exit code 1.
```

I wrapped the whole thing in double quotes, I'm not sure what else to do there.  Any ideas?

File: .github/workflows/deployNewHelp.yml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+jobs:
+  build:
+    env:
+      IS_PR_FROM_FORK: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.repo.fork }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
```
[the remark this answers is not in the document]
Comment (roryabraham) [reply .github/workflows/deployNewHelp.yml:33]: bump on this change. It makes the workflow much faster and is documented as a best practice here: https://github.com/Expensify/App/blob/44d09e58e73e937ad60e1c26107bfc022f5e4674/.github/workflows/README.md#fast-fetch

File: .github/workflows/deployNewHelp.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (roryabraham) [reply .github/workflows/deployNewHelp.yml:10]: If that's the case, can we remove the trigger than runs this workflow when code is pushed to main?

File: .github/workflows/deployNewHelp.yml [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+        env:
+          CF_API_KEY: ${{ secrets.CLOUDFLARE_TOKEN }}
+        working-directory: ./help  # Ensure cache purging is executed in /help
+
+      - name: Leave a comment on the PR
+        uses: actions-cool/maintain-one-comment@v3.2.0
+        if: ${{ github.event_name == 'pull_request' && env.IS_PR_FROM_FORK != 'true' }}
+        with:
+          token: ${{ secrets.OS_BOTIFY_TOKEN }}
```
Comment (roryabraham) [inline .github/workflows/deployNewHelp.yml:81]: NAB but generally it's a best practice to just use the `github.token` unless you have a reason it needs to be `OSBotify`. The built-in token has higher rate limits

**Review by roryabraham — CHANGES_REQUESTED** [review]
I think normally you could fix that ShellCheck error by just wrapping `"$HOME"` in quotes, but it looks like there's a bug with actionslint that makes it 🤮 if you try to do that. So we can fix this just by using the full path for the runner home, `/home/runner/`, like we do in other actions.

A few other things:

- Let's give the workflow a name
- we should simplify things and allow some more efficient caching by letting setup-ruby run the bundle install
- Let's add a `./help/.ruby-version` file to keep people's local version of ruby in sync with the version used in `deployNewHelp` and consolidate the source of truth for ruby version
- Let's remove `fetch-depth: 0` from the checkout. We don't need or want it
- there are a few other actions that are skipped for changes only in the `/docs` directory. i.e: we don't need to run app performance regression tests on a HelpDot PR. We should apply that change with the `/help` directory as well.
- We can run the pip install and cloudflare commands from root, the working dir doesn't matter for those.

Here's a full diff of the changes I'd like us to make before we merge this:

<details>
<summary>Full diff</summary>

```diff
diff --git a/.github/workflows/deployNewHelp.yml b/.github/workflows/deployNewHelp.yml
index 5262e246842..917ce1b3c0e 100644
--- a/.github/workflows/deployNewHelp.yml
+++ b/.github/workflows/deployNewHelp.yml
@@ -1,3 +1,5 @@
+name: Deploy New Help Site
+
 on:
   # Run on any push to main that has changes to the help directory
   push:
@@ -30,25 +32,16 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
 
-      # Set up Ruby and run commands inside the /help directory
+      # Set up Ruby and run bundle install inside the /help directory
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.3.4'  # Match your local Ruby version
           bundler-cache: true
-
-      - name: Install dependencies
-        run: |
-          bundle config set --local path 'vendor/bundle'
-          bundle install  # Install all gems, including custom plugins
-        working-directory: ./help  # Make sure dependencies are installed inside /help
+          working-directory: ./help
 
       - name: Build Jekyll site
-        run: |
-          bundle exec jekyll build --source ./ --destination ./_site  # Build within /help
+        run: bundle exec jekyll build --source ./ --destination ./_site
         working-directory: ./help  # Ensure Jekyll is building the site in /help
 
       - name: Deploy to Cloudflare Pages
@@ -64,20 +57,17 @@ jobs:
       - name: Setup Cloudflare CLI
         if: env.IS_PR_FROM_FORK != 'true'
         run: pip3 install cloudflare==2.19.0
-        working-directory: ./help  # Run CLI setup in /help
 
       - name: Purge Cloudflare cache
         if: env.IS_PR_FROM_FORK != 'true'
-        run: "$HOME/.local/bin/cli4 --verbose --delete 'hosts=[\"newhelp.expensify.com\"]' /zones/:9ee042e6cfc7fd45e74aa7d2f78d617b/purge_cache"
-
+        run: /home/runner/.local/bin/cli4 --verbose --delete hosts=["newhelp.expensify.com"] /zones/:9ee042e6cfc7fd45e74aa7d2f78d617b/purge_cache
         env:
           CF_API_KEY: ${{ secrets.CLOUDFLARE_TOKEN }}
-        working-directory: ./help  # Ensure cache purging is executed in /help
 
       - name: Leave a comment on the PR
         uses: actions-cool/maintain-one-comment@v3.2.0
         if: ${{ github.event_name == 'pull_request' && env.IS_PR_FROM_FORK != 'true' }}
         with:
-          token: ${{ secrets.OS_BOTIFY_TOKEN }}
+          token: ${{ github.token }}
           body: ${{ format('A preview of your New Help changes have been deployed to {0} :zap:️', steps.deploy.outputs.alias) }}
 
diff --git a/.github/workflows/preDeploy.yml b/.github/workflows/preDeploy.yml
index 79646817027..bfe860e6022 100644
--- a/.github/workflows/preDeploy.yml
+++ b/.github/workflows/preDeploy.yml
@@ -4,7 +4,7 @@ name: Process new code merged to main
 on:
   push:
     branches: [main]
-    paths-ignore: [docs/**, contributingGuides/**, jest/**, tests/**]
+    paths-ignore: [docs/**, help/**, contributingGuides/**, jest/**, tests/**]
 
 jobs:
   typecheck:
diff --git a/.github/workflows/reassurePerformanceTests.yml b/.github/workflows/reassurePerformanceTests.yml
index d4a25a63952..fb7a34d6fa0 100644
--- a/.github/workflows/reassurePerformanceTests.yml
+++ b/.github/workflows/reassurePerformanceTests.yml
@@ -4,7 +4,7 @@ on:
   pull_request:
     types: [opened, synchronize]
     branches-ignore: [staging, production]
-    paths-ignore: [docs/**, .github/**, contributingGuides/**, tests/**, '**.md', '**.sh']
+    paths-ignore: [docs/**, help/**, .github/**, contributingGuides/**, tests/**, '**.md', '**.sh']
 
 jobs:
   perf-tests:
diff --git a/.github/workflows/sendReassurePerfData.yml b/.github/workflows/sendReassurePerfData.yml
index 42d946cece9..884182bfc89 100644
--- a/.github/workflows/sendReassurePerfData.yml
+++ b/.github/workflows/sendReassurePerfData.yml
@@ -3,7 +3,7 @@ name: Send Reassure Performance Tests to Graphite
 on:
   push:
     branches: [main]
-    paths-ignore: [docs/**, contributingGuides/**, jest/**]
+    paths-ignore: [docs/**, help/**, contributingGuides/**, jest/**]
 
 jobs:
   perf-tests:
diff --git a/help/.ruby-version b/help/.ruby-version
new file mode 100644
index 00000000000..a0891f563f3
--- /dev/null
+++ b/help/.ruby-version
@@ -0,0 +1 @@
+3.3.4
```

</details>

File: .github/workflows/deployNewHelp.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (quinthar) [reply .github/workflows/deployNewHelp.yml:12]: Good point; I've removed it, and we'll see if it works when you merge.

**Review by roryabraham — DISMISSED** [review]
Approving because the push trigger is there on the existing `deployExpensifyHelp.yml` workflow, so NAB I suppose. But I still don't really get why it's there 😄

File: .github/workflows/deployNewHelp.yml [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      # Set up Ruby and run bundle install inside the /help directory
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
```
Comment (roryabraham) [inline .github/workflows/deployNewHelp.yml:41]: Whoops, I think that last commit removed the working-directory here by mistake:

```suggestion
          bundler-cache: true
          working-directory: ./help
```

**Review by roryabraham — CHANGES_REQUESTED** [review]

File: .github/workflows/deployNewHelp.yml (hunk 5 above)
Comment (quinthar) [reply .github/workflows/deployNewHelp.yml:41]: oh whaaaat

**Review by roryabraham — APPROVED** [review]

**Pull request merged**